### ISSUE TITLE
Error Message

### ISSUE THREAD

**Issue by ANRCO2021** [PirateNetwork/PirateWallet-Lite#25]
My Wallet will not sync.  I get this error

There was an error sending the transaction.  The error was:

Error fetching block Status (code: Unknown, message: "error reading a body from connection: broken pipe:}

**Comment by ANRCO2021** [comment]
I am panicking alittle, maybe alot.... I loaded up my wallet today with 4 accounts, I left, came home and saw they had all arrived but then I was froze up and now I am unable to sync or do anything.  My error message is above.  Please help

**Comment by ANRCO2021** [comment]
Renamed wallet then restarted my Wallet, entered the seed, it started the sync at 152855, I could see "No Connection" appear periodically in the bottom left corner.  Not sure what's going on as it worked great today.  I checked my Firewall settings, tweaked a couple fo things.  All other apps requiring Internet work fine.  I will keep trying to find new things to try but as it stands, I def need some help.  Thanks

**Comment by teoman002** [comment]
Hey, I am not a dev but I can help you.

Let us first collect the information about your situation:

- Which Operating System are you using?
- Did you write down your personal recovery word list during wallet creation? Have it ready in a safe location to recover your wallet!
- Did you change any files or parameters of the wallet?

You could try this:
- Use another operating system on the same computer, ubuntu-live on usb or dvd?
- Use another computer to open the lite wallet
- Use the recovery phrase to recover your wallet on a different system

If you were successful to import the wallet into Pirate-Wallet-lite, migrate to Pirate-Wallet with local blockchain:

- Open the menue File->Export all private keys
- Save the spending key in a text file
- Open the full blockchain pirate application, maybe you download the bootstrap files first, to reduce the time to sync.
- Open the menue File->Import Spending key

**Comment by ANRCO2021** [comment]
Hey thanks for responding

- I use Windows 7
- Yes, have word list
- No changes
- Um?  I'll put this try on the shelf
- Had thought of this as well however the only other computer I have is an old laptop with XP.  Was thinking I will go buy a new one today
- Will table all the rest until I try a different computer.

The transfer did work yesterday as I saw all the deposits on my screen but then I closed the wallet and when it reopened all the problems started.  Old technology is likely a factor.
